uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/PublicHoliday.php

```diff
@@ -27,15 +27,37 @@ public static function create($params) {
   }
 
   /**
+   * Delete a PublicHoliday with given ID.
+   * 
+   * @param int $id
+   */
+  public static function del($id) {
+    $publicHoliday = new CRM_HRLeaveAndAbsences_DAO_PublicHoliday();
+    $publicHoliday->id = $id;
+    $publicHoliday->find(true);
+    $publicHoliday->delete();
+  }
+
+  /**
+   * Return an array containing properties of Public Holiday with given ID.
+   * 
+   * @param int $id
+   * @return array|NULL
+   */
+  public static function getDefaultValues($id) {
```

Comment:
As @omarabuhussein suggested some time ago, this method name doesn't give a good idea of what it really does. The name was changed to getValuesArray in AbsencePeriod and WorkPattern (but it's still getDefaultValues in AbsenceType, probably where you looked). Can you rename it to getValuesArray?